www/include/monitoring/comments/commentHost.php

```diff
@@ -39,37 +39,37 @@
 if (!isset($oreon)) {
     exit();
 }
+$contactId = filter_var(
+    $_GET["contact_id"] ?? $_POST["contact_id"] ?? 0,
+    FILTER_VALIDATE_INT
+);
+$select = $_GET["select"] ?? $_POST["select"] ?? [];
+$form = new HTML_QuickFormCustom('Form', 'post', "?p=" . $p);
 
-    isset($_GET["contact_id"]) ? $cG = $_GET["contact_id"] : $cG = null;
-    isset($_POST["contact_id"]) ? $cP = $_POST["contact_id"] : $cP = null;
-    $cG ? $contact_id = $cG : $contact_id = $cP;
-
-    $form = new HTML_QuickFormCustom('Form', 'post', "?p=".$p);
-
-    /*
-	 * Path to the configuration dir
-	 */
-    $path = "./include/monitoring/comments/";
+/*
+ * Path to the configuration folder
+ */
+$path = "./include/monitoring/comments/";
 
-    /*
-	 * PHP functions
-	 */
-    require_once "./include/common/common-Func.php";
-    require_once "./include/monitoring/comments/common-Func.php";
-    require_once "./include/monitoring/external_cmd/functions.php";
+/*
+ * PHP functions
+ */
+require_once "./include/common/common-Func.php";
+require_once "./include/monitoring/comments/common-Func.php";
+require_once "./include/monitoring/external_cmd/functions.php";
 
 switch ($o) {
-    case "ah":
-        require_once($path."AddHostComment.php");
-        break;
-    case "dh":
-        DeleteComment("HOST", isset($_GET["select"]) ? $_GET["select"] : array());
-        require_once($path."viewHostComment.php");
-        break;
-    case "vh":
-        require_once($path."viewHostComment.php");
-        break;
-    default:
-        require_once($path."viewHostComment.php");
-        break;
+case "ah":
```

Comment:
indentation was good since it is in a switch statement

Reply:
indeed ':(
I'll correct it ASAP :+1: